### PR TITLE
Add survey link

### PR DESCRIPTION
**Pull request by johngruen**
TODO:

- [x] Hardcode an end date (sometime later December, maybe Thursday, December 21)
- [x] Put in a close/dismiss button
- [x] Use localStorage so that each user sees the survey request only once (doesn't matter if they dismiss or follow the link or ignore the whole thing)

### CODE REVIEW

**Comment by johngruen** [comment]
@ianb okay i rebased that PR

**Comment by johngruen** [comment]
@ianb button is there but non-functional

**Comment by ianb** [comment]
I'll take over from here

**Comment by ianb** [comment]
This will only show the survey link once. To see it again, in the console on a shot page, run `localStorage.hasSeenSurveyLink = ""`

It only shows on owner pages, and only to people who have English somewhere in their locales (doesn't have to be the primary language). It is hardcoded to expire at midnight (local time I think?) on December 21st.

File: server/src/pages/shot/controller.js [hunk 1]
```diff
@@ -9,6 +9,42 @@ const { shotGaFieldForValue } = require("../../ab-tests.js");
...
+    return false;
+  }
+  let foundEnglish = false;
+  // model.userLocales always contains some form of English because it's a server
+  // fallback. But navigator.languages does not have a fallback (except what the
+  // user indicates in their browser preferences)
+  if (model.userLocales[0].startsWith("en")) {
+    foundEnglish = true;
+  }
+  for (let locale of navigator.languages) {
```
Comment (chenba) [inline server/src/pages/shot/controller.js]: `foundEnglish = foundEnglish || navigator.languages.some(...);`?

**Comment by johngruen** [comment]
Let's change the string to: `Help choose which features we add next by taking this quick survey.`

**Pull request merged**